src/screens/Profile.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  Alert, Dimensions, StyleSheet, Platform, Text,
+} from 'react-native';
+import { LinearGradient } from 'expo';
+
+// galio component
+import { Block, Button, NavBar } from 'galio-framework';
+import theme from '../theme';
+
+const { height, width } = Dimensions.get('window');
+
+class Profile extends React.Component {
+  handleChange = (name, value) => {
+    this.setState({ [name]: value });
+  };
+
+  render() {
```

Comment:
@kostimarko let us know when you finish the [Profile screen](https://raw.githubusercontent.com/galio-org/galio/design/Hacktoberfest%20resources/PNGs/02%20Profile%20Screen.png), it should look like the one in design